apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1366,11 +1396,12 @@ namespace MWMechanics
 
         const MWMechanics::NpcStats& victimStats = victim.getClass().getNpcStats(victim);
 
-        // Simple check for who attacked first: if the player attacked first, a crimeId should be set
+        // Simple check for who attacked first:
+        // if the player or his follower/summon attacked first, a crimeId should be set.
         // Doesn't handle possible edge case where no one reported the assault, but in such a case,
         // for bystanders it is not possible to tell who attacked first, anyway.
         if (victimStats.getCrimeId() != -1)
-            commitCrime(attacker, victim, MWBase::MechanicsManager::OT_Murder);
+            commitCrime(attackerIsPlayerFollowerOrSummon ? player : attacker, victim, MWBase::MechanicsManager::OT_Murder);
```

Comment:
I'm not talking about a difference between summons/normal followers, but the fact that *any* follower attacking first doesn't produce a bounty for the player.

I suppose that this is a) intended to work this way (kinda makes sense that the player isn't responsible for the follower's actions, if they acted on their own accord) or b) no one really considered this case since followers usually don't attack a peaceful NPC on their own, except maybe for scripted events.